prime-router/metadata/fhir_mapping/hl7/codesystem/CodingSystemMapping.yml

```diff
@@ -4529,11 +4529,6 @@
   url: "http://terminology.hl7.org/CodeSystem/v2-0430"
 - id: "encounter-modeOfArrival-extension"
   url: "http://hl7.org/fhir/StructureDefinition/encounter-modeOfArrival"
-
-- id: "source-processing-id-system"
-  description: "Underlying Master Code System for V2 table 0103 (Processing ID)"
-  url: "https://terminology.hl7.org/3.1.0/CodeSystem-v2-0103.html"
-  oid: "urn:oid:2.16.840.1.113883.18.40"
```

Comment:
What's the rationale for removing this? I'm all for removing overloads but need to confirm

Reply:
It is no longer needed. That data is now in some resource property.